SwiftUIDZ/ContentView.swift

```diff
@@ -7,16 +7,131 @@
 
 import SwiftUI
 
+// Экран кошелька
 struct ContentView: View {
+
+    private enum Constants {
+        static let emptyText = ""
+        static let depositButtonName = "Пополнить баланс"
+        static let yesText = "Да"
+        static let depositActionSheetTitle = "Вы хотите пополнить баланс счета 1 на 5 единиц?"
+        static let depositAlertTitle = "Счет пополнен"
+        static let openAccountButtonName = "Открыть счет"
+        static let transferButtonButtonName = "Перевод средств"
+        static let settingsToggleName = "Показать настройки"
+        static let transferButtonAlertQuestion = "Вы хотите перевести 5 единиц со cчета 1 на cчет 2"
+        static let transferButtonAlertDeny = "Недостаточно средств на счете 1 или не открыт cчет 2"
+        static let balanceText = "Баланс счета"
+        static let equalText = "равен:"
+        static let animationSettingNumber = 0.6
+        static let zeroNumber = 0
+        static let oneNumber = 1
+        static let twoNumber = 2
+        static let defaultAccountCount = 1
+        static let defaultPointNumber = 5
+        static let cornerRadiusNumber: CGFloat = 10
+        static let zeroCGFloat: CGFloat = 0
+        static let twohundredCGFloat: CGFloat = 200
+    }
+    
+    @State private var isOnToggle = false
+    @State private var isDepositButtonPress = false
+    @State private var isUserAgree = false
+    @State private var isTransfer = false
+    @State private var balance = Constants.zeroNumber
+    @State private var bankAccountCount = Constants.defaultAccountCount
+    @State private var balanceAccounts = [Constants.zeroNumber]
+    @State private var accountFrom = Constants.emptyText
+    
     var body: some View {
         VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundColor(.accentColor)
-            Text("Hello, world!")
+            ZStack {
+                HStack{
+                    VStack {
+                        setupDepositButton()
+                        setupOpenAccountButton()
+                        setupTransferButton()
+                    }.padding()
+                    Spacer()
+                }
+                setupAccountBackground()
+                setupBankAccount()
+            }
+            setupSettingsToggle()
         }
         .padding()
+        .animation(.spring(response: Constants.animationSettingNumber,
+                           dampingFraction: Constants.animationSettingNumber,
+                           blendDuration: Constants.animationSettingNumber))
+    }
+    
+    private func setupDepositButton() -> some View {
```

Comment:
```suggestion
    private var depositButtonView: some View {
```
Так же без параметров в вычисляемое ниже так же поправь